build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -104,7 +127,10 @@ public static BuildTargetResult CopyCombinedFrameworkHostLayout(BuildTargetConte
             string sharedHostPublishRoot = c.BuildContext.Get<string>("SharedHostPublishRoot");
             Utils.CopyDirectoryRecursively(sharedHostPublishRoot, combinedRoot);
 
-            c.BuildContext["CombinedFrameworkHostRoot"] = combinedRoot;
+            string hostFxrPublishRoot = c.BuildContext.Get<string>("HostFxrPublishRoot");
+            Utils.CopyDirectoryRecursively(hostFxrPublishRoot, combinedRoot);
```

Comment:
We have two sets of these utilties apparently. 
Not a big deal but I think it'd be better to standardize on FS.cs